### PR TITLE
Editorial changes towards CG-DRAFT

### PR DESCRIPTION
**Pull request by csarven**
(An ocean of) editorial changes (correction class 1-2).

Change formatting / indenting to your liking. I tried to keep the diffs to minimal.

Will follow with a separate (small) PR for parts pertaining to CG-DRAFT proposal.

---

[Preview](http://htmlpreview.github.io/?https://github.com/solid/webid-profile/blob/e96c5d25404640329c3947e3f831a8a4bbe79a0b/index.html) | [Diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fraw.githubusercontent.com%2Fsolid%2Fwebid-profile%2F684fb2096e885602137b7d5fc6349c35cf388738%2Findex.html&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fsolid%2Fwebid-profile%2Fe96c5d25404640329c3947e3f831a8a4bbe79a0b%2Findex.html)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,324 +1,404 @@
...
+              </dd>
 
-            <dd id="Timea-Turdean">
-              <span about="" rel="schema:editor"><span about="https://timea.solidcommunity.net/profile/card#me"
-                  typeof="schema:Person"><a rel="schema:url" href="https://timea.solidcommunity.net/profile/card#me"><span
-                      about="https://timea.solidcommunity.net/profile/card#me" property="schema:name"><span
-                        property="schema:givenName">Timea</span>
-                      <span property="schema:familyName">Turdean</span></span></a></span></span>
-            </dd>
+              <dd id="Timea-Turdean">
```
Comment (VirginiaBalseiro) [inline index.html:332]: Should @csarven be under Editors as well?

Comment (csarven) [reply index.html:332]: Whatever, really, at this point. Added https://github.com/solid/webid-profile/pull/116/commits/01da96607146d705b98cf40a6ca909e054a9f9da

File: index.html [hunk 2]
```diff
@@ -1,324 +1,407 @@
...
```
Comment (VirginiaBalseiro) [inline index.html:332]: @timea-solid should be moved to Former Editors, as she stepped down from editing on October 2023.

Comment (csarven) [reply index.html]: Updated https://github.com/solid/webid-profile/pull/116/commits/7205cb732d837bbcbaaf7a9b184112831a5f4fe6

**Comment by csarven** [comment]
@jeff-zucker , can you please have a quick look over? It mostly brings the document up to date with random stuff. No change to normative content.

**Review by VirginiaBalseiro — APPROVED** [review]

**Review by jeff-zucker — APPROVED** [review]
The document looks good from a copy-editor standpoint.  I'll go ahead and approve this and submit some PRs and Issues on the wording and  normative changes I will suggest soon.

**Comment by VirginiaBalseiro** [comment]
Thanks @jeff-zucker . You are of course welcome to make PRs but they won't be included in the CG-DRAFT because we want to PR and publish soon. You can continue making contributions to the ED and those can be included in a future version.

**Comment by VirginiaBalseiro** [comment]
> It was not in any WebID Profile meeting I was aware of. 

Jeff, as you know, WebID-Profile group has not been meeting in favor of using time in the CG calls and having meetings ad hoc if needed, which hasn't happened. This was discussed in chat and in meetings over a year ago. 

> Who is "we" - all of the editors other than me? When was it decided? 

The CG. We have been discussing CG-DRAFT for all our documents for the past 3-4 weeks.

> without any consultation with me 

The PR is a consultation, and the PR for CG-DRAFT is another one.

> or response to issues I have raised, or opportunity to raise ones I have long been holding until there is a document for me to react to?

I don't understand what you mean that you have been "holding" until there "a document for you to react to". This document we're editing is the document you could react to, create issues, or make PRs. In the past year or so I have had problems even getting my PRs reviewed, so I am a little bit surprised to hear you say that you haven't been asked for your response. 

Do you wish to object to WebID-Profile specification to be published as CG-DRAFT? If so please make your reasons known and we can discuss it.

**Comment by csarven** [comment]
There may be a middle ground. Let's continue in https://github.com/solid/webid-profile/pull/117 and there are some related considerations for the CG, e.g., whether to keep editor's draft and CG-DRAFT as separate documents. General discussion around that is also covered in https://github.com/solid/specification/issues/587

**Comment by jeff-zucker** [comment]
I was under  the impression that you were preparing an extensive use case document.  Previously you had objected  to my introduction and my suggested changes and said that the use cases would answer my objections.    I never saw any major changes to the document and assumed you had simply stopped work on it to focus on your many other duties.  No, I did not respond to your PRs because none of them addressed my concerns.  I brought up some of my concerns in https://github.com/solid/webid-profile/issues/110 which got two positive responses from the community and zero responses of any kind from other editors of the document.  Since I did not get a response to that, I did not submit furthyer ones, assuming that work was stalled and would be addressed later.  Given this history, it is unbelievable to me that you tell me you are publishing a complete document and there is no time to wait for me to respond.   I will submit PRs later this week.  If you publish it before then, remove my name because it will be published over my objection.

**Comment by jeff-zucker** [comment]
I do not want to hold up the draft over issue #110  , that will take a longer discussion.  But there are other changes I need to propose related to consistently distinguishing Solid Profile from Solid WebID Profile document and to the mentions of issuers and indexes.  I do not expect them to be major or contentious but would like an opportunity to suggest them.

**Comment by csarven** [comment]
W3C documentation may not be clear out there but CG-DRAFTs can be revised indefinitely. I see absolutely no reason why the document (irrespective to being a CG-DRAFT or not) can't incorporate any changes. In the PR I've also mentioned how an ED like can be worked on in parallel to whatever CG-DRAFT or TR/ release is out there. Can we continue this discussion in that PR?

**Comment by csarven** [comment]
Hmm, I think it shouldn't matter much. Perhaps you can do it off the main branch for now and rebase if/when PR 117's contents are available from the main branch. I'm not sure if that's particularly different than branching off `feature/cg-draft` (from PR 117) and staying synchronised. (I'd go for main and rebasing.. because I'm no git master..)

**Comment by VirginiaBalseiro** [comment]
> I was under the impression that you were preparing an extensive use case document.

Indeed the [use cases](https://github.com/solid/webid-profile/blob/main/notes/use-cases.md) that was taken as a work in parallel to the Solid WebID Profile specification, and if we want to publish it as a CG-DRAFT, it would need its own authors and editors. We have a list of use cases in the Introduction: https://solid.github.io/webid-profile/#use-cases covering a range of use cases, and that was initially brought up through in this PR: https://github.com/solid/webid-profile/pull/87 with the understanding that it covered https://github.com/solid/webid-profile/issues/52 and https://github.com/solid/webid-profile/issues/55 .

> I never saw any major changes to the document and assumed you had simply stopped work on it to focus on your many other duties.

All changes can be tracked in for example https://github.com/solid/webid-profile/pulls?q=is%3Apr+is%3Aclosed and https://github.com/solid/webid-profile/commits/main/ .
 
Here are all of the changes since your last commit: https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fraw.githubusercontent.com%2Fsolid%2Fwebid-profile%2Fe5c629010fcdb44d56b5773f978e6d1c6e2cb658%2Findex.html&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fsolid%2Fwebid-profile%2F7205cb732d837bbcbaaf7a9b184112831a5f4fe6%2Findex.html

**Pull request merged**